clients/src/main/java/org/apache/kafka/common/errors/OffsetMetadataTooLarge.java

```diff
@@ -19,17 +19,10 @@
 
     private static final long serialVersionUID = 1L;
 
-    public OffsetMetadataTooLarge() {
```

Comment:
Is it OK to remove constructors from existing exceptions? I would have preferred if constructors like this one had not been added, but now that they're there I think we have to deprecate first.